main.py

```diff
@@ -183,156 +184,154 @@ def compose_message(issues):
-        except ValueError:
-            reply.description = "I can't give you un-natural issues." + \
-                " I'm not a monster!!"
-            await channel.send(embed=reply)
-            return
-        if number_of_issues > 10:
-            number_of_issues = 10
-            reply.description = "Let's curb that enthusiasm.. just a little"
-            await channel.send(embed=reply)
-        await give_random(channel, number_of_issues)
-        return
+`!feedback <N> issue/pr`
+Retrieves N least-recently-updated issue/pr in the feedback queue
 
-    elif command == "q" or command == "review":
-        if len(args) != 1:
-            await channel.send(embed=reply)
-            return
-        try:
-            number_of_issues = int(args[0].strip())
-            if number_of_issues < 1:
-                raise ValueError("Positive integer needed")
-        except ValueError:
-            await channel.send("This queue is 100% natural. Check your orders")
-            return
-        if number_of_issues > 10:
-            number_of_issues = 10
-            reply.description = "Let's curb that enthusiasm.. just a little"
-            await channel.send(embed=reply)
-
-        author = message.author
-        if command == "q":
-            author = None
-        elif isinstance(author, discord.Member):
-            author = author.nick or author.name
-        elif isinstance(author, discord.User):
-            author = author.name
-
-        await review_q(channel, number_of_issues, author)
-        return
-    return
+`!fq <N> issue/pr`
+Retrieves N least-recently-updated issue/pr in the feedback queue'''
+	await channel.send(embed=reply)
+
+
+@bot.command(name='rand')
+async def rand_cmd(ctx, number_of_issues, noun, *):
+    await check_length_issues(number_of_issues)
+    git_config=read_config("command_config.json")["rand"]
+    update_config(noun, git_config)
+
+    await queue_handler(channel=bot.channel, number_of_issues=number_of_issues, author=None, git_config=git_config)
+
+
+@bot.command(name='review')
+async def review_cmd(ctx, number_of_issues, noun, *):
+    await check_length_issues(number_of_issues)
+    git_config=read_config("command_config.json")["review"]
+    update_config(noun, git_config)
+
+    author = ctx.author
+    elif isinstance(author, discord.Member):
+        author = author.nick or author.name
+    elif isinstance(author, discord.User):
+        author = author.name
+
+    await queue_handler(channel=bot.channel, number_of_issues=number_of_issues, author=author, git_config=git_config)
+
+
+@bot.command(name='rq'):
+async def rq_cmd(ctx, number_of_issues, noun, *):
+    await check_length_issues(number_of_issues)
+    git_config=read_config("command_config.json")["review"]
+    update_config(noun, git_config)
+
+    await queue_handler(channel=bot.channel, number_of_issues=number_of_issues, author=None, git_config=git_config)
+
+
+@bot.command(name='feedback')
+@bot.command(name='fq')
```

Comment:
We need to test this

Reply:
Feel free to test it. You need to create a [new application](https://discord.com/developers/)

Reply:
On it.